cf-check/dump.c

```diff
@@ -483,6 +492,11 @@ int dump_main(int argc, const char *const *const argv)
         {
             mode = DUMP_PORTABLE;
         }
+        else if (StringMatchesOption(option, "--tskey", "-t"))
+        {
+            tskey_filename = argv[offset];
+            offset += 1;
```

Comment:
The message below about "Only one option supported" isn't accurate anymore is it. Change it.

Reply:
@olehermanse do you think cf-check should use --tskey/-t option as a single option which forces DUMP_NICE? e.g. keep only one option situation with dump command?

Reply:
I'll keep as single option defaulting to DUMP_NICE. Easy and makes some sense since it assumes nice strings and structs.